vcpkg/TOC.yml

```diff
@@ -56,6 +56,10 @@
         href: users/examples/versioning.getting-started.md
       - name: Create a x-script Asset Caching source for NuGet
         href: examples/asset-caching-source-nuget.md
+    - name: Experimental features

```

Comment:
I think this section should be just `GitHub Integration` or something like that. We have other experimental features in other pages

Reply:
There are some specific reasons I was thinking of adding this. I'll follow up in an email thread.

Reply:
Let's put this under the "Consuming Libraries" section for now.

Reply:
After some offline discussion, I'm going to try this at the documentation root and see what that looks like. If it's weird, I'll either put it in the "Consuming Libraries" section for now or see if an "Integrations" root folder makes it better.